excel_xml_export.php

```diff
@@ -52,6 +52,11 @@
 require_api( 'print_api.php' );
 require_api( 'utility_api.php' );
 
+if( !access_has_project_level( config_get( 'export_issues_threshold' ) ) ) {
+	# current user is not allowed to export content => redirection to print page (compliant with bug_count = 0)
+	print_header_redirect( 'view_all_set.php?type=0&print=1' );
```

Comment:
A redirect is the workaround used to not display a blank page.
The destination page is the same that is done later on the same page :

	if( 0 == $p_bug_count ) {
		print_header_redirect( 'view_all_set.php?type=0&print=1' );
	}

I copy the same destination link, but I'm pretty sure that the "&print=1" part shoudn't be present in both links. 

Just tested on mantisbt server : use the "Excel Export" option when filter return no bug is redirected to print_all_bug_page.php. Curious Behaviour.